vars/users.yaml

```diff
@@ -2,8 +2,11 @@
 create_users: true
 
 account_names:
-  - { user: 'elim', key: 'https://github.com/limed.keys' }
-  - { user: 'jwhitlock', key: 'https://github.com/jwhitlock.keys' }
-  - { user: 'rjohnson', key: 'https://github.com/escattone.keys' }
-  - { user: 'jmize', key: 'https://github.com/jgmize.keys' }
-  - { user: 'dparfitt', key: 'https://github.com/metadave.keys' }
+  # Web SRE Team
+  # According to According to https://github.com/orgs/mozilla-it/teams/it-se/members?query=membership:child-team
+  - { user: 'duallain', key: 'https://github.com/duallain.keys' }
```

Comment:
should we keep escattone's key in here?

jwhitlock still works at moz, but is probably safe to remove.  everyone else should for sure be deleted.

Reply:
sure, good idea!

Reply:
👍 to removing that @jwhitlock guy